### ISSUE TITLE
Empty sequences to be removed

### ISSUE THREAD

**Issue by peterdesmet** [inbo/mica-occurrences#19]
Two invalid sequences were included in export:

```
782bfb18-e820-469c-acda-bf2c1b6f575d:seq,701956d1-c5f9-49f4-a42a-6f5f7fa77943,782bfb18-e820-469c-acda-bf2c1b6f575d,,+02:00,unclassified,motion detection,False,,,,,,,,,,,,
aa985039-918b-42d8-ab5e-bc8700736048:seq,a0441d90-467f-4e8b-875e-3e275ebccf53,aa985039-918b-42d8-ab5e-bc8700736048,,+02:00,unclassified,motion detection,False,,,,,,,,,,,,
```

**Comment by peterdesmet** [comment]
The above two no longer occur, but these ones do:

```
71d10e66-8189-452b-b911-b45560fa5221,380077a1-389f-4ddb-bd6d-9c5e40ce1c56,208d0088-57fe-4314-9949-4de1b3cc0fcd,,+02:00,unknown,False,,,,,,,,,human,,,,,,,,
1438520c-68e9-4927-91fb-4794d2570b05:seq,32011df8-9770-470c-abe5-c5afcd68b0e3,1438520c-68e9-4927-91fb-4794d2570b05,,+01:00,unclassified,motion detection,False,,,,,,,,,,,,
25e5417e-cd3b-4c57-ae19-319f3bf67a41,d247656d-cb58-4fb1-b860-7e7f863646ee,fe0dfaae-d65c-4320-bbce-4df960bdec94,,+01:00,blank,False,,,,,,,,,human,,,,,,,,
501aac62-c647-44d4-8904-daf352e3fc46:seq,f34058ce-aba3-4599-b1e3-b41e5a4739e6,501aac62-c647-44d4-8904-daf352e3fc46,,+01:00,unclassified,False,,,,,,,,,,,,,,,,,
d27a6b66-f3c6-47e5-8da9-cdb427fa8d58:seq,f34058ce-aba3-4599-b1e3-b41e5a4739e6,d27a6b66-f3c6-47e5-8da9-cdb427fa8d58,,+01:00,unclassified,False,,,,,,,,,,,,,,,,,
```

**Comment by peterdesmet** [comment]
Issues are either resolved or covered in #23
